openwisp_users/tests/test_admin.py

```diff
@@ -256,3 +256,24 @@ def test_admin_add_superuser_org_not_required(self):
         user = queryset.first()
         self.assertTrue(user.is_superuser)
         self.assertTrue(user.is_staff)
+
+    # def test_operator_change_user_permissions(self):
+    #     operator = self._create_operator()
+    #     operator.is_staff = True
+    #     # print(operator._meta.get_fields())
+    #     operator.user_permissions.set(Permission.objects.all())
+    #     self.client.force_login(operator)
+    #     admin = self._create_admin()
+    #     response = self.client.get(reverse('admin:openwisp_users_user_change', args=[admin.pk]))
+    #     html = '<input type="checkbox" name="is_superuser"'
+    #     self.assertNotContains(response, html)
+
+    def test_user_add_user(self):
+        operator = self._create_operator()
+        self.client.force_login(operator)
+        operator.user_permissions.set(Permission.objects.all())
+        operator_permission = Permission.objects.get(codename='_has_add_permission')
+        operator.user_permissions.remove(operator_permission)
+        # print(Permission.objects.all())
+        response = self.client.get(reverse('admin:openwisp_users_user_add'))
+        self.assertNotContains(response, '<input type="text" name="username"')
```

Comment:
configure your editor to always add new lines at the end of files, notice the last red character here on github